docker/nginx/conf.d/server/server.skylink

```diff
@@ -6,7 +6,12 @@ include /etc/nginx/conf.d/include/init-optional-variables;
 
 location / {
     set_by_lua_block $skylink { return string.match(ngx.var.host, "%w+") }
-    set $path $uri;
+    set_by_lua_block $path {
+        -- strip ngx.var.request_uri from query params - this is basically the same as ngx.var.uri but
+        -- do not use ngx.var.uri because it will already be unescaped and we need to use escaped path
+        -- examples: escaped uri "/b%20r56+7" and unescaped uri "/b r56 7"
+        return string.gsub(ngx.var.request_uri, "?.*", "")
```

Comment:
I trust we don't have to escape the '?' here right.

Reply:
`string.gsub(ngx.var.request_uri, "?.*", "")` will strip the uri from `?` and anything after that so:

`/hello?world=foo&bar=baz` will become `/hello`